src/features/admin/components/InvalidAdmin.tsx

```diff
@@ -0,0 +1,12 @@
+import { Markdown } from "~/components/ui/Markdown";
+
+export function InvalidAdmin() {
+  return (
+    <div>
+      <Markdown>{`### Invalid Admin Account
+  Only admins can access this page.
+
+  `}</Markdown>
```

Comment:
I don't think we need markdown here

Reply:
Using Markdown here as CSS is not configured to distinguish styles for headings or paragraphs, example using just HTML 
<img width="1639" alt="image" src="https://github.com/privacy-scaling-explorations/maci-rpgf/assets/578688/97231279-bb63-4bf4-a248-35dc9971b073">

Reply:
There is a `Heading` component and you can use it with different styles.